client/servicediscovery/tso_service_discovery.go

```diff
@@ -160,6 +171,7 @@ func NewTSOServiceDiscovery(
 		primaryURL:    "",
 		secondaryURLs: make([]string, 0),
 		urls:          make([]string, 0),
+		version:       0,
```

Comment:
When the TSO client starts up, does it use version=0 for the initial request?

Furthermore, if this initial request is routed to a TSO service with a lower version, is that behavior expected?

Reply:
No, this version just works for requesting the keyspace group. Most updates don't relate to this client's keyspace ID, so it doesn't need to be updated.

Reply:
The higher version of the server side doesn't mean that the client's keyspace group has changed.